force-app/main/default/classes/NKS_TemporaryAddressController.cls

```diff
@@ -169,4 +183,27 @@ public with sharing class NKS_TemporaryAddressController {
     private static String validateString(String stringToCheck) {
         return String.isBlank(stringToCheck) ? null : stringToCheck.left(255);
     }
+
+    @TestVisible
+    private static void setValues(List<TemporaryAddress> addresses, String personIdent) {
```

Comment:
Disse to metodene kunne vel i teorien blitt gjort generiske og bare vært definert ett sted hvis adresse-propertiene matchet (de burde vel matchet uansett?)

Reply:
Jeg tenkte også å ha bare en metode, men vi har to forskjellige objekter. skal dobbeltsjekke om det finnes muligheten for å gjøre den generisk

Reply:
Tror du bare kan bruke SObject